packages/expo-stories/server/generateId.ts

```diff
@@ -0,0 +1,16 @@
+import crypto from 'crypto';
+
+// TODO - figure out the best way to generate a pure id from filepath
+//  needs to be a string with valid JS characters
+function generateId(filePath: string) {
```

Comment:
I would rather say the process you're looking for is `hashing`, but I don't have a suggestion how to do it right 🤔

Reply:
> Another round of questions/remarks:
> 
> * why are you using `*.stories.*` instead of `*.story.*`?

this is partly convention, partly because multiple stories for a given component are expected to be exported from the file

> * do we want to add `expo-stories/build` to the repo?

not sure about this yet - the server code does for sure, the rest probably not - ill think about this deeper

> * do we want to treat `expo-stories` as publicly exposed module? I wonder if we're going to maintain and publish it like any other module in `packages/*` or rather do we want to use it internally? If the latter I suggest moving it out the `packages` directory as we have many scripts that perform some generic actions on every `packages`' subdirectory.

the original idea was a developer facing package - at this point its just for internal use

> * this idea might be considered premature optimisation, but maybe we can extract `expo-stories/shared` directory containing components to completely separate package and name it `expo-components-library` (or similar, like `ant design`, `material UI`, etc). My idea is to create a library delivering components for every native app we're maintaining like `test-suite`, `bare-expo` (that might not be so representative), `ncl` (if it's going to survive this whole storyfying process 😅)

agreed on this, also what I was thinking

> * in `package.json#expoStories` I suggest using the very same syntax the original `package.json` is proposing, so instead of `extraNodeModules` let's go with `dependencies` - why to add another confusion while you use it exactly as `dependencies` in the server/cli commands code?

ive since removed the use of `extaNodeModules` in lieu of `packages` but I can change this to `dependencies` if that's clearer

> * why do we have `expo-stories/clients/app`? Are you planning to add more client apps?

that's right - I have in mind a web app that will be able to interact with story apps

thanks again for the feedback!